gui/ui/retracerdialog.ui

```diff
@@ -23,104 +23,28 @@
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="sizeConstraint">
+    <enum>QLayout::SetFixedSize</enum>
+   </property>
    <item>
-    <layout class="QHBoxLayout" name="horizontalLayout">
-     <item>
-      <spacer name="horizontalSpacer">
-       <property name="orientation">
-        <enum>Qt::Horizontal</enum>
-       </property>
-       <property name="sizeType">
-        <enum>QSizePolicy::Fixed</enum>
-       </property>
-       <property name="sizeHint" stdset="0">
-        <size>
-         <width>20</width>
-         <height>20</height>
-        </size>
-       </property>
-      </spacer>
-     </item>
+    <layout class="QVBoxLayout" name="checkboxesLayout">
+     <property name="leftMargin">
+      <number>27</number>
```

Comment:
preferably number which could be divided by two

Reply:
I set this to 27 so that pixel-for-pixel this matched what previously achieved with a horizontal spacer.
I can certainly adjust this though.

Reply:
Honestly, not having this custom at all would look better, imho. So - making the left margin 0 (default).
I'll get you a screenshot of what that looks like as soon as I figure out this weird text elision bug.
the & in `Run & Check` is being elided to `Run ... Check` for some reason, when it shouldn't be.
The next commit you see is not intended to be committed, but is there to illustrate the issue in a Qt forum post.

I'll let you know when I think this is ready.

Reply:
Here's what it looks like without a left margin:

![image](https://github.com/apitrace/apitrace/assets/156712/163d4908-7155-4218-be10-a9c7d29ec3e3)

Reply:
It looks more minimalistic; I like it.